src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -216,6 +216,7 @@ private String getIdForLabel(Label label) {
     private Pod getPodTemplate(KubernetesSlave slave, Label label) {
         final PodTemplate template = getTemplate(label);
         String id = getIdForLabel(label);
+        String shouldAlwaysPullImage = template.isAlwaysPullImage() ? "Always" : "IfNotPresent";
```

Comment:
actually I would get rid of the var and just use it below

`.withImagePullPolicy(template.isAlwaysPullImage() ? "Always" : "IfNotPresent")`